lib/public/Settings/SetDeclarativeSettingsValueEvent.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace OCP\Settings;
```

Comment:
```suggestion
namespace OCP\Settings\Event;
```

suggestion for a better location. many other lib/public/* have a dedicated namespace for event classes